docs/user/canvas.asciidoc

```diff
@@ -138,15 +131,13 @@ To access your saved elements, click *Add element > My elements*.
 
 Add a panel that you saved in *Visualize Library* to your workpad. 
 
-. Click *Add element > Add from {kib}*.
-
-. Select the panel you want to add.
+. Click *Add from library*, then select the panel you want to add.
 
 . To use the customization options, open the panel menu, then select one of the following options:
 
 * *Edit map* &mdash; Opens <<maps,Maps>> so that you can edit the panel.
 
-* *Edit visualization* &mdash; Opens the visualization editor so that you can edit the panel.
+* *Edit Visualization* &mdash; Opens the visualization editor so that you can edit the panel.
```

Comment:
If this uses title caps in the UI, it should be changed to sentence caps:  Edit visualization

Reply:
When we do a UI review on Canvas, let's include this as one of our changes.